examples/flatten_video_tracks.cpp

```diff
@@ -0,0 +1,74 @@
+#include "util.h"
+
+#include <opentimelineio/stackAlgorithm.h>
+#include <opentimelineio/timeline.h>
+
+#include <iostream>
+#include <sstream>
+
+namespace otio = opentimelineio::OPENTIMELINEIO_VERSION;
+
+int main(int argc, char** argv)
+{
+    if (argc != 3)
+    {
+        std::cout << "Usage: flatten_video_tracks (inputpath) (outputpath)" << std::endl;
+        return 1;
+    }
+    
+    // Read the file
+    PythonAdapters adapters;
```

Comment:
This is a cool thing, but I'd suggest breaking out a separated "embedding a python interpreter" example, rather than using it as a utility in cpp in general. I can imagine people starting to proliferate python embeddings via copy-pasta, not understanding whether that's a requirement of OTIO or not. Reading and writing a JSON file via C++ is supported out of the box, so in terms of being instructive, the latter is the approach I'd recommend.

Reply:
In the past we've encouraged people to execute `otioconvert` or `otiocat` as sub-processes to avoid the subject of embedding Python in their application. Example programs that show how to accomplish either or both of these would be very helpful. Then folks can pick whichever suits their use case.

Reply:
Also, using `otiocat` (or equivalent code via embedded Python) would avoid the need for a temp file when reading. You could probably avoid the temp file when writing also.

Reply:
Sure thing, I'll make a dedicated example like you suggest and the other examples will just work with JSON files.

Reply:
Thanks Josh, I'll checkout otioconvert/otiocat, I wasn't thrilled with the temp file approach.